src/utils/aws/ses.py

```diff
@@ -0,0 +1,127 @@
+"""Create DNS email records from SES."""
+# Third-Party Libraries
+import boto3
+
+route53 = boto3.client("route53")
+ses = boto3.client("ses")
+
+
+def list_hosted_zones(names_only: bool = False):
+    """
+    List hosted zones.
+
+    Set names_only to true if only hosted zone names are needed.
+    """
+    if not names_only:
+        return route53.list_hosted_zones()["HostedZones"]
+
+    return [hosted_zone.get("Name") for hosted_zone in list_hosted_zones()]
```

Comment:
This function is already created in src/utils/aws/clients.py. That's where all of our aws clients are. I would probably move these functions over there as well.